server/index.js

```diff
@@ -8,17 +8,44 @@ const db = require('../db/index.js');
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-app.use(express.static('../public'));
+app.use(express.static(`${__dirname}/../public`));
 
-// Request Handling
+/* Request Handling */
 
-app.get('/api/item/:id', (req, res) => {
-  let id = req.params.id;
-  db.Products.findAll({
+// Get initial current product accessories
+app.get('/items/accessories/ipad', (req, res) => {
+  db.Accessories.findAll({
     where: {
-      id: id
-    }
-  });
+      categoryName: 'appleTablets',
+    },
+  })
+    .then((results) => {
+      res.status(200).json(results);
+    })
+    .catch((error) => {
+      console.log('There was an error getting accessories from the DB: ', error);
+      res.sendStatus(404);
```

Comment:
A+ on always sending a response, even when there's an error.
I personally prefer sending a status of `5xx` ("sorry, we screwed up on the server side") over `4xx` ("sorry, you screwed up on the client side").